<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionsWidget</class>
 <widget class="QWidget" name="OptionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="3" column="0">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>ports INI-file:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>chanels INI-file:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QFEnhancedLineEdit" name="edtChannels"/>
   </item>
   <item row="3" column="1">
    <widget class="QFEnhancedLineEdit" name="edtPorts"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>timing INI-file:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QFEnhancedLineEdit" name="edtTiming"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QFEnhancedLineEdit</class>
   <extends>QLineEdit</extends>
   <header>qfenhancedlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
